src/ResourceManager/Resources/Commands.Resources/ManagementPartner/RemoveManagementPartner.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Management.Automation;
+using Microsoft.Azure.Management.ManagementPartner;
+using PartnerResources = Microsoft.Azure.Commands.Resources.Properties.Resources;
+
+namespace Microsoft.Azure.Commands.Resources
+{
+    [Cmdlet(VerbsCommon.Remove, "AzureRmManagementPartner", SupportsShouldProcess = true)]
+    public class RemoveManagementPartner : AzureManagementPartnerCmdletsBase
+    {
+        [Parameter(Position = 0, Mandatory = true, HelpMessage = "The management partner id")]
+        [ValidateNotNull]
+        public string PartnerId { get; set; }
+
+        public override void ExecuteCmdlet()
+        {
+            if (ShouldProcess(string.Format(PartnerResources.RemoveManagementParnterTarget, PartnerId),
+                string.Format(PartnerResources.RemoveManagementParnterAction, PartnerId)))
+            {
+                AceProvisioningManagementPartnerApiClient.Partner.DeleteAsync(PartnerId).Wait();
+                WriteObject(string.Format(PartnerResources.RemovedManagementPartner, PartnerId));
```

Comment:
Returning a string is never preferred for a cmdlet. In this case, you'll want to implement [-PassThru](https://gist.github.com/cormacpayne/568f454127bd39805796958f2ac90a18#returning-no-output). Then, this will conditionally return a bool (true on success and throws on failure) when the -PassThru switch is provided.